src/index.js

```diff
@@ -15,13 +18,27 @@ function streamToBuffer(stream) {
 }
 
 class CyclicS3FS {
-  constructor(bucketName, config={}) {
+  constructor(bucketName='', config={}) {
     this.bucket = bucketName
     this.config = config
     this.s3 = new S3Client({...config});
+    this.isLocal = !bucketName
```

Comment:
should do this based on no AWS creds I think 
and/or another flag in the config object 

in which case probably should be more explicit about what config goes into instantiating the s3 client 
AWS region, etc